lib/commands/cache.js

```diff
@@ -161,14 +161,9 @@ class Cache extends BaseCommand {
   // npm cache add <tarball>...
   // npm cache add <folder>...
   async add (args) {
-    const usage = 'Usage:\n' +
-      '    npm cache add <tarball-url>...\n' +
-      '    npm cache add <pkg>@<ver>...\n' +
-      '    npm cache add <tarball>...\n' +
-      '    npm cache add <folder>...\n'
     log.silly('cache add', 'args', args)
     if (args.length === 0)
-      throw Object.assign(new Error(usage), { code: 'EUSAGE' })
+      throw this.usageError('First argument to `add` is required')
```

Comment:
I opted to change this error message to be more consistent with other commands.

Here's a diff of the error messages since we aren't capturing it in a snapshot:
```diff
diff --git a/oldoutput.log b/newoutput.log
index 974aaaa34..9693e7641 100644
--- a/oldoutput.log
+++ b/newoutput.log
@@ -1,10 +1,25 @@
 npm ERR! code EUSAGE
+npm ERR! 
+npm ERR! Usage: First argument to `add` is required
+npm ERR! 
+npm ERR! npm cache
+npm ERR! 
+npm ERR! Manipulates packages cache
+npm ERR! 
 npm ERR! Usage:
-npm ERR!     npm cache add <tarball-url>...
-npm ERR!     npm cache add <pkg>@<ver>...
-npm ERR!     npm cache add <tarball>...
-npm ERR!     npm cache add <folder>...
+npm ERR! npm cache add <tarball file>
+npm ERR! npm cache add <folder>
+npm ERR! npm cache add <tarball url>
+npm ERR! npm cache add <git url>
+npm ERR! npm cache add <name>@<version>
+npm ERR! npm cache clean [<key>]
+npm ERR! npm cache ls [<name>@<version>]
+npm ERR! npm cache verify
+npm ERR! 
+npm ERR! Options:
+npm ERR! [--cache <cache>]
 npm ERR! 
+npm ERR! Run "npm help cache" for more info
```